instrumentation/google.golang.org/grpc/otelgrpc/config.go

```diff
@@ -55,10 +58,23 @@ func newConfig(opts []Option) *config {
 	c := &config{
 		Propagators:    otel.GetTextMapPropagator(),
 		TracerProvider: otel.GetTracerProvider(),
+		MeterProvider:  global.MeterProvider(),
 	}
 	for _, o := range opts {
 		o.apply(c)
 	}
+
+	c.meter = c.MeterProvider.Meter(
+		instrumentationName,
+		metric.WithInstrumentationVersion(SemVersion()),
+		metric.WithSchemaURL(semconv.SchemaURL),
+	)
+	var err error
+	// the unit of rpc.server.duration is milliseconds
+	if c.rpcServerDuration, err = c.meter.SyncInt64().Histogram("rpc.server.duration"); err != nil {
```

Comment:
If the unit is known it should be passed as an option using [`WithUnit`](https://pkg.go.dev/go.opentelemetry.io/otel/metric@v0.32.3/instrument#WithUnit).